src/mantine-demos/src/demos/form/Form.demo.zod.tsx

```diff
@@ -66,7 +66,8 @@ const schema = z.object({
 
 function Demo() {
   const form = useForm({
-    schema: zodResolver(schema),
+    asyncResolver: true,
+    schema: zodResolver(schema) as any,
```

Comment:
API does not fell nice. Can you make it as before `schema: zodResolver(schema)`, without `asyncResolver` field and `as any`?